packages/block-editor/src/components/link-control/index.js

```diff
@@ -260,11 +261,27 @@ function LinkControl( {
 				/>
 			) }
 
-			<LinkControlSettingsDrawer
-				value={ value }
-				settings={ settings }
-				onChange={ onChange }
-			/>
+			<div className="block-editor-link-control__tools">
+				<LinkControlSettingsDrawer
+					value={ value }
+					settings={ settings }
+					onChange={ onChange }
+				/>
+				{ onRemove &&
+					isFunction( onRemove ) &&
+					value &&
+					! isEditingLink &&
+					! isCreatingPage && (
+						<Button
+							className="block-editor-link-control__unlink"
+							isDestructive
+							variant="link"
+							onClick={ onRemove }
+						>
+							{ __( 'Unlink' ) }
```

Comment:
Probably have to be `_x` here 😅

Reply:
So I understand better, why do we need to use `_x` here particularly? Other instances in this file use `__` so I'm curious.

Reply:
I looked it up here

https://developer.wordpress.org/reference/functions/_x/

But I'm still not sure.

Reply:
From what I understood, `_x` allows us to provide context to texts that are often ambiguous, especially for short words like `Unlink`. `Unlink` could mean different things depending on the context. I guess it would only matter if there're other appearances of the same word though 🤔. Just a nitpick here, don't need to do anything 😅.